bin/deploy.sh

```diff
@@ -160,14 +173,23 @@ deploy_charts()
     else
         VALUE_OVERIDE=""
```

Comment:
I realize this is not your error, could could you please correct the spelling of OVERRIDE in this variable name throughout the script?